src/CarbonAware.WebApi/src/Controllers/CarbonAwareController.cs

```diff
@@ -204,6 +204,58 @@ public IActionResult BatchForecastData(IEnumerable<EmissionsForecastBatchDTO> re
         return Ok(result);
     }
 
+    /// <summary>
+    /// Retrieves the actual data that occurred between the time boundaries and calculates the carbon intensity that occurred during that window. 
+    /// </summary>
+    /// <remarks>
+    ///  This endpoint is useful for reporting the measured carbon intensity for a specific time period in a specific location.
+    /// </remarks>
+    /// <param name="location"> the location in </param>
+    /// <param name="jobStartTime"> the time at which the job and thus carbon usage starts </param>
+    /// <param name="jobEndTIme"> the time at which the job and thus carbon usage ends </param>
+    /// <returns> a single object that contains the location, time boundaries and carbon intensity value </returns>
+    /// <response code="200">Returns the a single object that contains the info about the request and the marginal carbon intensity </response>
+    /// <response code="400">Returned if any of the requested items are invalid</response>
+    /// <response code="500">Internal server error</response>
+    [Produces("application/json")]
+    [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(IEnumerable<>))]
+    [ProducesResponseType(StatusCodes.Status400BadRequest, Type = typeof(ValidationProblemDetails))]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError, Type = typeof(ValidationProblemDetails))]
```

Comment:
I noticed that we dropped the 501 compared to the last one.  Was that a purposeful choice or a copy/paste choice?  I am fine with either outcome, but prefer we make purposeful choices :)

Reply:
purposeful choice since it should always have actual values and never not be implemented